x11rb/src/x11_utils.rs

```diff
@@ -94,10 +97,8 @@ macro_rules! atom_manager {
         #[derive(Debug)]
         $(#[$cookie_meta])*
         $vis struct $cookie_name<'a, C: $crate::protocol::xproto::ConnectionExt> {
-            phantom: std::marker::PhantomData<&'a C>,
-            $(
-                $field_name: $crate::cookie::Cookie<'a, C, $crate::protocol::xproto::InternAtomReply>,
-            )*
+            __private_phantom: std::marker::PhantomData<&'a C>,
+            __private_cookies: Vec<$crate::cookie::Cookie<'a, C, $crate::protocol::xproto::InternAtomReply>>,
```

Comment:
```suggestion
            __private_cookies: ::std::vec::Vec<$crate::cookie::Cookie<'a, C, $crate::protocol::xproto::InternAtomReply>>,
```

Just in case a potential user defines their own `Vec` struct.

Reply:
Thanks for catching that! I turned the existing `tests/x11_utils.rs` into a test case for that and also found a `std::` which should be `::std::`.